hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/codec/OMDBDefinition.java

```diff
@@ -230,14 +229,14 @@ String.class, new StringCodec(), OmKeyInfo.class,
           SnapshotInfo.class,
           new OmDBSnapshotInfoCodec());
 
-  public static final DBColumnFamilyDefinition<String, OmKeyRenameInfo>
+  public static final DBColumnFamilyDefinition<String, String>
```

Comment:
```suggestion
  // SnapshotRenamedKeyTable that complements the keyTable (or fileTable) entries
  // of the immediately previous snapshot in the same snapshot scope (bucket or volume)
  public static final DBColumnFamilyDefinition<String, String>
```

Reply:
Also it would be useful to put a larger block comment here to demonstrate the intended usage of this table.

Can asciify the tables in the design doc here.

Reply:
Sure, will add it.